### ISSUE TITLE
Use Storage Access Framework on Android

### ISSUE THREAD

**Issue by MisterY** [keybase/client#22126]
Considering that Keybase keeps encrypted files online and a local secure copy on a device, it would probably make sense to utilize [Storage Access Framework](https://developer.android.com/guide/topics/providers/document-provider) on Android.
This way, the Keybase client would be able to offer files to other applications. 
Many cloud clients nowadays implement this, which is very convenient.

**Comment by MisterY** [comment]
This would cover #22063

**Comment by fortepc** [comment]
Yes please! I would love to use Keepass, and need a way to keep my devices in sync, and using KBFS would be the ideal way to do it.
